.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Hook Docs
+
+on:
+ push:
+   branches:
+    - develop
```

Comment:
Let's change this to `master` so that the hook docs don't update until we actually are ready to release a new version

Reply:
@helen should we make a similar change to Distributor and ElasticPress?